packages/rollup/src/index.js

```diff
@@ -105,6 +105,10 @@ async function loadConfigFile(configFile) {
 
   await runRollup(configFile, inputOptions, outputOptions);
 
+  // Ensure node isn't caching a previous version of the config file
+  // https://github.com/rollup/rollup/blob/v1.31.0/cli/run/loadConfigFile.ts#L52
+  delete require.cache[cjsConfigFile];
```

Comment:
Should this be `require.cache[require.resolve(cjsConfigFile)]` to ensure it matches the cache key?

Reply:
Because node probably does the same thing before caching? I think that would make sense, I can add `require.resolve` where we define `cjsConfigFile`...

Reply:
Actually the file doesn't exist until this line (after `runRollup` creates it above) so I've updated to exactly what you suggested here.